mbuild/packing.py

```diff
@@ -133,8 +166,16 @@ def fill_box(compound, n_compounds=None, box=None, density=None, overlap=0.2,
         compound = [compound]
     if n_compounds is not None and not isinstance(n_compounds, (list, set)):
         n_compounds = [n_compounds]
+
     if not isinstance(fix_orientation, (list, set)):
         fix_orientation = [fix_orientation]*len(compound)
+    try:
+        iter(fix_orientation[0])
+    except:
```

Comment:
I think you might want this to be `except TypeError:`

Reply:
Well in this case I am using the exception to trigger a warning about the change in API for `fix_orientation`, its not necessaryily an error, because if a non-iterable value is passed into `fix_orientation` then it defaults to the old behaviour of constraining rotation about all or none of the axes.

Perhaps there might be a better way to do this, but as is, if a user passes `fix_orientation=True` they will get the warning message which explains they now have an option to fix rotations about each of the x,y,z axes, and using `True` will behave the same as passing `fix_orientation=(True, True, True)`.